modules/main-rules/pages/transportation.adoc

```diff
@@ -1,59 +1,79 @@
 = Transportation
 
 On some unit data sheets you'll see that the unit has one of two special abilities that relate to transportation: {transport-x} or {hitch-x}.
-We explain these and other special abilities in more detail elsewhere; but transportation is a vital aspect of the Movement phase so we'll discuss it here too.
 
-When we speak of these transportation abilities, we use the terms 'embark/embarked' and 'disembark/disembarked' to describe actions and states that relate to the units that you transport.
+These abilities indicate units that can carry -- transport -- other units across the battlefield.
+These abilities have some features in common, but also differ in some important ways.
 
-Both the {transport-x} and {hitch-x} abilities indicate units that can carry -- transport -- other units across the battlefield.
-But they work in slightly different ways.
+== Embark and disembark
```

Comment:
Just adding a note here to repeat something I mentioned in the main thread, so I don't lose track of it:

It may yet be best to integrate the embark/disembark placement conditions into each set of transportation rules, rather than have them under a separate heading — I find it a bit disjointed.

Reply:
I'll resolve this conversation, as it is now outdated by my latest commit to address this.